features/discovery/src/main/java/org/opennms/netmgt/discovery/actors/Discoverer.java

```diff
@@ -66,13 +73,29 @@
     private static final Logger LOG = LoggerFactory.getLogger(Discoverer.class);
 
     private final PingerFactory m_pingerFactory;
+    
+    private LocationAwarePingSweepClient client;
 
     public Discoverer(final PingerFactory pingerFactory) {
         m_pingerFactory = Preconditions.checkNotNull(pingerFactory, "pingerFactory argument");
     }
 
     public DiscoveryResults discover(DiscoveryJob job) {
```

Comment:
Discovery will need more changes if we want to properly incorporate the PingSweep RPC client, simply changing it here is not the right approach.